test/Sleep-test.js

```diff
@@ -0,0 +1,91 @@
+import { expect } from 'chai';
+import Sleep from '../src/Sleep';
+
+const User = require('../src/User')
+
+describe('Sleep', () => {
+  it('should be a function', () => {
+    expect(Sleep).to.be.a('function')
+  });
+
+  it('should be an instance of Sleep', () => {
```

Comment:
I don't think we need this becuase it's obvious, or can be combined with the function it statement above. 

 Up to you though No harm :)